packages/react-composites/src/composites/CallComposite/components/ExpandedLocalVideoTile.tsx

```diff
@@ -65,10 +67,12 @@ const containerStyles: IStackStyles = {
 };
 
 const darkenedOverlayStyles: (palette: IPalette, isVideoReady: boolean) => IStyle = (palette, isVideoReady) => {
+  const colorWithAlpha = (isVideoReady ? '#000000' : palette.neutralLight) + '60';
```

Comment:
This can't be safe because we can't assume how `palette.neutralLight` is specified.

Reply:
Done.